selfdrive/ui/qt/sidebar.cc

```diff
@@ -43,9 +43,17 @@ Sidebar::Sidebar(QWidget *parent) : QFrame(parent) {
   setFixedWidth(300);
 
   QObject::connect(uiState(), &UIState::uiUpdate, this, &Sidebar::updateState);
+
+  pm = std::make_unique<PubMaster, const std::initializer_list<const char *>>({"userFlag"});
 }
 
 void Sidebar::mouseReleaseEvent(QMouseEvent *event) {
+  if (home_btn.contains(event->pos())) {
+    // TODO: maybe only create event while car started?
```

Comment:
this is alright as is, since loggerd doesn't run offroad anyway. this solution is temporary until we have some real design for event flagging.

Reply:
ah okay!